EIPS/eip-1.mediawiki

```diff
@@ -13,46 +13,43 @@ EIP stands for Ethereum Improvement Proposal. A EIP is a design document providi
+The EIP process begins with a new idea for Ethereum. It is highly recommended that a single EIP contain a single key proposal or new idea. Small enhancements or patches often don't need an EIP and can be injected into the Ethereum development workflow with a patch submission to the corresponding Ethereum issue tracker. The more focused the EIP, the more successful it tends to be. The EIP editor reserves the right to reject EIP proposals if they appear too unfocused or too broad. If in doubt, split your EIP into several well-focused ones.
 
-Each EIP must have a champion -- someone who writes the EIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The EIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is EIP-able. Posting to the  the [https://forum.ethereum.org/categories/protocol-and-client-discussion Protocol Discussion] forum or opening an [https://github.com/ethereum/EIP/issues Issue] is the best way to go about this.
+Each EIP must have a champion -- someone who writes the EIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea.
 
-Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used.
+Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [https://www.reddit.com/r/ethereum/ the Ethereum subreddit] and [https://gitter.im/ethereum/ one of the Ethereum Gitter chat rooms].
```

Comment:
> And the benevolent dictator phase only lasts while the EIP is a mere issue, as it should.

I don't think any EIPs have been submitted as drafts since people started using issues for them, though. Or are you talking about how it might/should work once fixed?

> Rather than going full RFC, how about a preEIP repo set up to do a better job.technically, while maintaining the automatic integration with Github and Gitter?

I definitely support the idea of having a separate draft stage, though I think it can be part of the same repo. My suggestion (copied from Skype) for a process goes something like this:

 1. Author writes draft EIP in their own repo, with arbitrary number of revisions and changes.
 2. Once author is happy it’s reasonably stable, author submits draft EIP to main repo. Review process is relatively lightweight, and it gets accepted as a draft.
 3. Revisions to the draft are submitted as PRs, each with their own discussion.
 4. When the draft is stable (at the editor’s discretion), a PR to migrate it to an official numbered EIP is made. Any remaining issues with the draft must be addressed before that PR is merged.
 5. Once it’s official and numbered, substantial changes aren’t allowed - they must be submitted as new EIPs that obsolete the old one.